content/en/docs/concepts/containers/images.md

```diff
@@ -184,7 +184,7 @@ These options are explained in more detail below.
 Specific instructions for setting credentials depends on the container runtime and registry you chose to use. You should refer to your solution's documentation for the most accurate information.
 
 For an example of configuring a private container image registry, see the
-[Pull an Image from a Private Registry](/docs/tasks/configure-pod-container/pull-image-private-registry)
+[Pull an Image from a Private Registry](/docs/tasks/kubelet-credential-provider/kubelet-credential-provider)
```

Comment:
why changing this link?

Reply:
Again, from the PR: https://github.com/kubernetes/website/pull/37920/files

But it makes sense to keep this link as-is because private registry image pull is a different thing.